src/html/index.html.template

```diff
@@ -63,6 +63,9 @@
       import "<%= latestCoreJS %>";
       import "<%= latestAppJS %>";
       import "<%= latestHassIconsJS %>";
+      {% for extra_module in extra_modules -%}
```

Comment:
All imports are resolved prior to the script being executed. In this case that's bad because if one import would fail, we would not set the path to customPanelJS and things like hassio panel wouldn't work! So let's add an if check if extra_modules has entries, and if so print a new `<script type="module">` tag below this tag.

Alternatively, we can also just put the `window.customPanelJS` line in a separate type="module" tag above this one, that way it's always set.

For ES5 this is fine as imports are not resolved prior to code being executed.

Reply:
Does that mean that one failed import would stop all others too?

Reply:
I _think_ so.

Reply:
better idea. Just print a script tag with a `src` for each file. 

```html
<script type="module" crossorigin="use-credentials" src="{{ extra_module}}"></script>
```

Reply:
👍